ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py

```diff
@@ -290,3 +290,29 @@ def test_tensorflow_logical_xor(
         x=np.asarray(x[0], dtype=input_dtype[0]),
         y=np.asarray(x[1], dtype=input_dtype[1]),
     )
+
+    
+# negative
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_tf.valid_float_dtypes),
```

Comment:
We can use `ivy_tf.valid_numeric_dtypes` as this function should be able to accept integer dtypes aswell as float

Reply:
Fixed that @jkeane508

Reply:
@jkeane508 
I should also add uint8, uint16, uint32, and uint64 to the unsupported types of negative right?

Reply:
@kanakmi
Yep, makes sense :)